distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-admin-console-contents.adoc

```diff
@@ -0,0 +1,156 @@
+[cols="1,4", options="header"]
+|===
+|Configuration
+|The Configuration tab lists all bundles associated with the application as links to configure any configurable properties of that bundle.
+
+|Details
+|The Details tab gives a description, version, status, and list of other applications that either required for, or rely on, the current application.
+
+|Features
+|The features tab breaks down the individual features of the application that can be installed or uninstalled as configurable features.
+
+|===
+
+==== Managing Applications
+
+The *Manage* button enables activation/deactivation and adding/removing applications.
+
+===== Activating / Deactivating Applications
+
+The *Deactivate* button stops individual applications and any dependent apps.
+Certain applications are central to overall functionality and cannot be deactivated.
+These will have the *Deactivate* button disabled.
+Disabled apps will be moved to a list at the bottom of the page, with an enable button to reactivate, if desired.
+
+[IMPORTANT]
+====
+Deactivating the `platform-app`, `admin-app`, and `security-services-app` will cause errors within the system, so the capabilities to do so have been DISABLED.
+====
+
+===== Adding Applications
+
+The *Add Application* button is at the end of the list of currently active applications.
+
+===== Removing Applications
+
+To remove an application, it must first be deactivated.
+This enables the *Remove Application* button.
+
+===== Upgrading Applications
+
+Each application tile includes an *Upgrade* button to select a new version to install.
+
+==== System Settings Tab
+
+The configuration and features installed can be viewed and edited from the System tab as well, however, it is recommended that configuration be managed from the applications tab.
+
+[IMPORTANT]
+====
+In general, applications should be managed via the applications tab.
+Configuration via this page could result in an unstable system.
+Proceed with caution!
+====
+
+===== Managing Features Using the ${admin-console}
+
+. Select the appropriate application.
+. Select the *Features* tab.
+. Uninstalled features are shown with a *play* arrow under the *Actions* column.
+.. Select the *play* arrow for the desired feature.
+.. The *Status* will change from *Uninstalled* to *Installed*.
+. Installed features are shown with a *stop* icon under the *Actions* column.
+.. Select the *stop* icon for the desired feature.
+.. The *Status* will change from *Installed* to *Uninstalled*.
+
+===== Adding Feature Repositories
+
+If needed, custom feature repositories can be added to extend ${branding} functionality.
+
+. Select the *Manage* button in the upper right.
+. Select the *Add an Application* tile
+. Select *File Upload* to add a new `.kar` OR `.jar` file.
+. Select the *Maven URL* tab and enter the URL of the feature repository.
+.. Select the *Add URL* button.
+//TODO: add a maven URL
+. Select the *Save Changes* button.
+
+==== Configuring HTTP Port from ${admin-console}
```

Comment:
This is an important note, but it's in a strange place. Maybe we should have a more general section that describes all the optional features, like we have for the http to https proxy (next section), and include this note in that section?